arm-sql/2014-04-01/swagger/sql.core.json

```diff
@@ -2330,6 +2330,27 @@
           "format": "date-time",
           "description": "The recovery period start date of the Azure SQL Database. This records the start date and time when recovery is available for this Azure SQL Database (ISO8601 format)."
         },
+        "createMode": {
```

Comment:
It is not

---
In reply to: [85607302](https://github.com/Azure/azure-rest-api-specs/pull/643#discussion_r85607302) [](ancestors = 85607302)